### PR TITLE
Add advanced ratelimit example

### PR DESCRIPTION
**Pull request by RicHincapie**
## Description

Add advanced ratelimit example with per route actions.

Fixes https://github.com/istio/istio.io/issues/14263

## Reviewers

- [ ] Ambient
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [ ] Localization/Translation

### CODE REVIEW

**Comment by RicHincapie** [comment]
@kfaseela WDYT? local ratelimit test was set to work also

File: content/en/docs/tasks/policy-enforcement/rate-limit/index.md [hunk 1]
```diff
@@ -155,15 +159,70 @@ backend, is used below.
...
 
-### Local rate limit
+### Global rate limit advanced case
+
+This example uses regex to match `/api/*` `uri` and defines a rate limit action inserted at the route level using the VirtualService http name. The PATH value `api` inserted in the prior example comes into play.
+
+1. Delete the prefix `/api/v1/products` from bookinfo VirtualService:
+
+    {{< text bash >}}
+    $ kubectl patch vs bookinfo --type=json -p='[{"op": "remove", "path": "/spec/http/0/match/4"}]'
```
Comment (zirain) [inline content/en/docs/tasks/policy-enforcement/rate-limit/index.md:169]: can we simplify this? maybe just apply a new vs with same name?

File: content/en/docs/tasks/policy-enforcement/rate-limit/index.md [hunk 2]
```diff
@@ -310,38 +398,51 @@ EOF
...
 ### Verify local rate limit
 
 Although the global rate limit at the ingress gateway limits requests to the `productpage` service at 1 req/min,
-the local rate limit for `productpage` instances allows 10 req/min.
+the local rate limit for `productpage` instances allows 4 req/min.
 To confirm this, send internal `productpage` requests, from the `ratings` pod, using the following `curl` command:
 
 {{< text bash >}}
-$ kubectl exec "$(kubectl get pod -l app=ratings -o jsonpath='{.items[0].metadata.name}')" -c ratings -- curl -s productpage:9080/productpage -o /dev/null -w "%{http_code}\n"
+$ kubectl exec "$(kubectl get pod -l app=ratings -o jsonpath='{.items[0].metadata.name}')" -c ratings -- bash -c 'for i in {1..5}; do curl -s productpage:9080/productpage -o /dev/null -w "%{http_code}\n"; sleep 1; done'
```
Comment (zirain) [inline content/en/docs/tasks/policy-enforcement/rate-limit/index.md:429]: we can simplify this with `kubctl exec deploy/ratings`

**Review by zirain — APPROVED** [review]
lgtm

**Review by kfaseela — APPROVED** [review]

**Pull request merged**